pyclam/manifold.py

```diff
@@ -1121,7 +1115,6 @@ def __init__(
             **kwargs
     ):
         """ A Manifold needs the data from which to learn the manifold, and a distance function to use while doing so.
-
```

Comment:
Very weird seeing so many formatting changes in docstrings. I would like to see these changes reverted